docs/server/source/production-deployment-template/troubleshoot.rst

```diff
@@ -104,3 +104,28 @@ MongoDB instance correctly.
 
 More information about this configuration is provided in
 :doc:`this document <cloud-manager>`.
+
+6. Create a Persistent Volume from existing Azure disk storage Resource
+---------------------------------------------------------------------------
+The configuration file to create PV for Mongo Stateful Set is located at ``mongodb/mongo-pv.yaml``.
```

Comment:
You should say why they might want to do this, i.e. because there was a Kubernetes cluster which stored some MongoDB data on disks, and now you want to reuse those disks, maybe with a new MongoDB StatefulSet, or an entirely new Kubernetes cluster.